Documentation/sourcebuild-in-repos/new-repo.md

```diff
@@ -302,26 +304,53 @@ will detect and publish.
 
 [intermediate nupkg]: https://github.com/dotnet/source-build/blob/master/Documentation/planning/arcade-powered-source-build/intermediate-nupkg.md
 
-## Add Downstream Dependency
+## Incorporate the new repo into the Source-Build dependency tree
 
-The graph defined by `eng/Version.Details.xml` defines what is included
-in source build. This dependency graph starts at
+Once your repo can be source-built, it is time to register it into the
+Source-Build dependency tree. The graph of the product is defined by the
+`eng/Version.Details.xml` files. This dependency graph starts at
 [dotnet/installer](https://github.com/dotnet/installer/blob/main/eng/Version.Details.xml).
-The .NET source build insfrastructure walks this graph and determines
-the full set of repos to build. Source build will only build one copy/commit
-of each repo (the first commit found in the dependency graph). Therefore,
-when adding a new repositories, a dependency must be created within the
-`eng/Version.Details.xml` graph. To do this, go to the repo which depends on
-the new repo and add a
-[new source build dependency](#engversiondetailsxml) to the new source build
-repo.
+The dependendecies of repos declared in these files are walked and the
+first copy/commit of each repo found in the dependency graph is used.
+
+Therefore, when adding new repositories, a dependency must be created within
+the `eng/Version.Details.xml` graph. To do this, **go to the repo which depends
+on the new repo and add a [new source build dependency](#engversiondetailsxml)
+to the new source build repo**.
+
+## Source build repos and the VMR
+
+Another effect of adding a new source build repository is that its sources
+will be synchronized into the [Virtual Monolithic Repository of .NET](https://github.com/dotnet/dotnet).
+The VMR is then where the official Source-Build happens from. The sources
+are synchronized once the associated commit/package flows into `dotnet/installer`.
+
+In order for the sources of the new repo to by synchronized into the VMR,
+the repo needs to be registered in the
+[`source-mappings.json` file](https://github.com/dotnet/dotnet/blob/main/src/source-mappings.json)
+which tells the tooling where from and which sources should be synchronized.
+Please open a PR in [`dotnet/dotnet`](https://github.com/dotnet/dotnet) and
+add your repository. The name must match the name declared in the `SourceBuild`
+tag in `Version.Details.xml` created in the previous step.
```

Comment:
This is actually now a subject to change - we will be mirroring the `source-mappings.json` file from installer too once we complete https://github.com/dotnet/arcade/issues/11982

But we can modify this later.